### DISCUSSION TITLE
LDAP and local groups

### DISCUSSION THREAD

**Discussion by arp-mbender** [requarks/wiki#7484] [Help / Questions]
I'm trying to set up an internal knowledge base using LDAP as the login provider. The basics seem to work - I can log in, and if I've got a group in WikiJS and LDAP with the same name then WikiJS will detect the membership when the user signs in.

However I have two issues with this process:
- It seems I cannot create a new group for system admins and I cannot re-use the existing Administrators group because (LDAP naming takes priority!)
- If I manually assign a user to the default Administrators group then that assignment is reset when the assigned user logs in as WikiJS re-checks memberships

In other words - it seems WikiJS makes no distinction between LDAP groups and "local" groups.

a) Can I create a "local" group where I can manually assign LDAP accounts without fear of those accounts being automatically kicked off?
b) Can I create a new "admin" group with full privileges? I suppose I could manually change the permissions for the custom "admin" group in the database, but is there an official method?
